hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/S3SecretStore.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.hadoop.ozone.om;
+
+import org.apache.hadoop.ozone.om.helpers.S3SecretValue;
+
+import java.io.IOException;
+
+/**
+ * S3 secret store interface.
+ */
+public interface S3SecretStore {
+
+  /**
+   * Store provided s3 secret with associated kerberos ID.
+   * @param kerberosId kerberos ID.
+   * @param secret s3 secret.
+   * @throws IOException in case when secret storing failed.
+   */
+  void store(String kerberosId, S3SecretValue secret) throws IOException;
```

Comment:
Consistent naming is preferred,  either (store, get, revoke) or (storeSecret, getSecret, revokeSecret).  Consider OmMetadataManagerImpl implements the S3SecretStore. And there are many functions in OmMetadataManagerImpl. I more prefer (storeSecret, getSecret, revokeSecret) .